src/core/core.datasetController.js

```diff
@@ -290,6 +291,12 @@ helpers.extend(DatasetController.prototype, {
 	_update: function(reset) {
 		var me = this;
 		me._configure();
+
+		me._cachedDataOpts = null;
+		if (!me._areDataOptsDynamicallyResolved()) {
```

Comment:
What happens if a user changes one of these properties, say from funcA to funcB. Would this detect that?

Reply:
If any property is a `function` then `_areDataOptsDynamicallyResolved` would return `true`.